data/json/items/tool/misc.json

```diff
@@ -227,6 +227,24 @@
       "done_message": "You unroll the fur mat and lay it on the ground."
     }
   },
+  {
+    "id": "hose_garden",
+    "looks_like": "hose",
+    "type": "TOOL",
+    "name": { "str": "garden hose" },
+    "description": "This is a flexible garden hose.  If you cut it in smaller pieces, it could be used for crafting, or siphoning fuel from a vehicle.",
+    "weight": "1632 g",
+    "volume": "1500 ml",
+    "longest_side": "30 cm",
+    "price": 600,
+    "price_postapoc": 300,
+    "to_hit": -1,
```

Comment:
accuracy base: -2
grip: solid +1
length: short +1
surface: any +0
balance: neutral +0

should be to_hit +0 so you can just omit it.